src/reporting/DisputeBondToken.sol

```diff
@@ -101,4 +101,8 @@ contract DisputeBondToken is DelegationTarget, Typed, Initializable, ERC20Basic,
         bondHolder = _destinationAddress;
         return true;
     }
+
+    function totalSupply() public constant returns (uint256) {
+        return supply;
+    }
```

Comment:
You probably will want to delete this as well, and potentially the storage variable.

Reply:
Ah, I see.  This is necessary to get it to compile.  Keep this then and just have it return a hard-coded `1`, then remove the `supply` storage variable.

Reply:
Oops, I had forgotten I'd added this when I made my earlier comment about totalSupply.  Yeah, I'll just hard-code this to return 1.